mist-api/src/lib/util.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+// Vendor
+const _ = require('lodash')
+
+// Local
+const util = require('core-lib/util')
+
+exports.splitIntoQueryTerms = (value) => {
```

Comment:
Shouldn't it be splitIntoMatchAllWhereTerms since I use it for `$all` not `$any`? Or did I understand the meaning of `Anywhere` in `splitIntoMatchAnywhereTerms` wrong?

Reply:
What the user does with the resulting array is decoupled from this method. The consumer may use either `$all` or `$any`. I was referring to the fact that `%term%` means it could match anywhere in the target string.